fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -202,11 +208,22 @@ fun TabItem(
                 .then(widthModifier)
         ) {
             badgeWithIcon()
+            var fontSize = remember { mutableStateOf(14.sp) }
+            var textStyle by remember(textColor) { mutableStateOf(TextStyle(color = textColor, textAlign = TextAlign.Center, fontSize = fontSize.value)) }
+
             if (textAlignment == TabTextAlignment.VERTICAL) {
                 Spacer(modifier = Modifier.height(2.dp))
-                BasicText(
+                Text(
```

Comment:
Lets not use material related APIs. Any reason why Basic text is not used. basictext is compose API